main.rb

```diff
@@ -37,24 +37,36 @@ def options
   Please select a value:
   1) List all books
   2) List all labels
-  3) Add a label
-  4) Add a book
-  5) Exit'
+  3) List all genres
+  4) List all albums
+  5) Add a label
+  6) Add a genre
+  7) Add a book
+  8) Add an album
+  9) Exit'
   gets.chomp
 end
 
-def run(catalog)
+def run(catalog) # rubocop:disable Metrics/CyclomaticComplexity,Metrics/MethodLength
```

Comment:
Only to take notes for our team. In the refactor we will change to %w[ ] methodology you share with us.